<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteEdit</class>
 <widget class="QWidget" name="NoteEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Редактор</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цвет:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="color">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Зелёный</string>
   </property>
   <item>
    <property name="text">
     <string>Зелёный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Синий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Красный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тема:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="theme">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>52</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ярлыки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tags">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>108</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Категории:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cats">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Текст:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boldButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="italicButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="underlinedButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
